hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -885,7 +885,7 @@
 
 <property>
   <name>dfs.datanode.directoryscan.throttle.limit.ms.per.sec</name>
-  <value>1000</value>
+  <value>-1</value>
```

Comment:
LGTM. But the comment `1000 is the default setting` also needs to update.

Reply:
@tomscut Thanks for your review. The description is updated.